docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,34 +1,57 @@
 [[connect-to-elasticsearch]]
 == Add data to {kib}
 
-To start working with your data in {kib}, you can:
+The fastet way to add data to {kib} is to use one of the built-in options,
+available from the home page. You can:
 
-* Upload a CSV, JSON, or log file with the File Data Visualizer.
+* Add data from popular apps and services
+* Add data using an Elastic Agent
```

Comment:
Not exactly clear how line 7 is different from line 8 (Elastic Agent is the new way of adding data from popular apps and services, no?)

Reply:
I'd also indicate here that this is beta